### PR TITLE
built shared upload media function

### PR DESCRIPTION
**Pull request by Matt-Tan15**

### CODE REVIEW

File: src/shared/utils/UploadMedia.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Axios from "axios";
+
+const UploadMedia = async (files: FileList) => {
+  const formData = new FormData();
+  formData.append("file", files[0]);
+  formData.append("upload_preset", "ds2dkkvk");
+
+  try {
+    const data = await Axios.post(
```
Comment (raylu511) [inline src/shared/utils/UploadMedia.ts:9]: I would do something like `const { data: { version, publicId, format } }`

File: src/shared/utils/UploadMedia.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import Axios from "axios";
+
+const UploadMedia = async (files: FileList) => {
```
Comment (raylu511) [inline src/shared/utils/UploadMedia.ts:3]: we should camelCase the function name

Comment (Matt-Tan15) [reply src/shared/utils/UploadMedia.ts:3]: gotcha

File: src/shared/utils/UploadMedia.ts (hunk 1 above)
Comment (Matt-Tan15) [reply src/shared/utils/UploadMedia.ts]: That's much better, thanks!

**Review by raylu511 — APPROVED** [review]

**Review by DLiSWE — APPROVED** [review]

**Pull request merged**